Description: remove optional conscrypt depends
 Conscrypt is a library that is only really maintained as an internal
 dependency for Android.  BouncyCastle should work just fine, so no
 need to package up a large complicated library just for this.
Author: Hans-Christoph Steiner
Forwarded: not-needed
Last-Update: 2016-10-12
--- a/build/make/tools/signapk/src/com/android/signapk/SignApk.java
+++ b/build/make/tools/signapk/src/com/android/signapk/SignApk.java
@@ -32,7 +32,6 @@
 import org.bouncycastle.operator.OperatorCreationException;
 import org.bouncycastle.operator.jcajce.JcaContentSignerBuilder;
 import org.bouncycastle.operator.jcajce.JcaDigestCalculatorProviderBuilder;
-import org.conscrypt.OpenSSLProvider;
 
 import com.android.apksig.ApkSignerEngine;
 import com.android.apksig.DefaultApkSignerEngine;
@@ -1003,9 +1002,6 @@
     public static void main(String[] args) {
         if (args.length < 4) usage();
 
-        // Install Conscrypt as the highest-priority provider. Its crypto primitives are faster than
-        // the standard or Bouncy Castle ones.
-        Security.insertProviderAt(new OpenSSLProvider(), 1);
         // Install Bouncy Castle (as the lowest-priority provider) because Conscrypt does not offer
         // DSA which may still be needed.
         // TODO: Stop installing Bouncy Castle provider once DSA is no longer needed.
